src/evse_man.h

```diff
@@ -53,7 +55,7 @@ typedef uint32_t EvseClient;
 #define EVSE_VEHICLE_ETA    (1 << 2)
 
 #ifndef EVSE_MANAGER_MAX_CLIENT_CLAIMS
-#define EVSE_MANAGER_MAX_CLIENT_CLAIMS 10
+#define EVSE_MANAGER_MAX_CLIENT_CLAIMS 11
```

Comment:
Shouldn't need to increase the number of clims, this is the total number of claims that cn be made at any one time.

Reply:
Okay